server/src/main/java/io/druid/server/coordinator/rules/IntervalLoadRule.java

```diff
@@ -41,9 +42,9 @@ public IntervalLoadRule(
       @JsonProperty("tieredReplicants") Map<String, Integer> tieredReplicants
   )
   {
-    validateTieredReplicants(tieredReplicants);
+    this.tieredReplicants = tieredReplicants == null ? ImmutableMap.<String, Integer>of() : tieredReplicants;
```

Comment:
I have made changes to convert the null to use default values and not allow empty ones. Can you please check if not allowing empty values cause any problem with the legacy rules in your system.